seed_identity_store/utils.py

```diff
@@ -1,9 +1,13 @@
 from django.conf import settings
+from identities.models import ADDRESS_TYPES
 
 
 def get_available_metrics():
     available_metrics = []
     available_metrics.extend(settings.METRICS_REALTIME)
     available_metrics.extend(settings.METRICS_SCHEDULED)
 
+    for a_type in ADDRESS_TYPES:
+        available_metrics.append('identities.change.{}.sum'.format(a_type))
```

Comment:
Please add these in the settings file with the other metrics rather than here. An example of how we usually do it is here https://github.com/praekelt/hellomama-registration/blob/develop/hellomama_registration/settings.py#L249-L250